src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,149 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import './TodoList.scss';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+import classNames from 'classnames';
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+interface Props {
+  todos: Todo[],
+  selectUser: (userId: number) => void,
+}
+
+interface State {
+  titleToSearch: string,
+  completeStatus: string
+}
+
+export class TodoList extends React.Component<Props, State> {
+  state = {
+    titleToSearch: '',
+    completeStatus: '',
+  };
+
+  handleInputChange = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>,
+  ) => {
+    this.setState(prevState => ({
+      ...prevState,
+      [event.target.name]: event.target.value.toLowerCase(),
+    }));
+  };
+
+  preparedTodos = () => {
+    const { titleToSearch, completeStatus } = this.state;
+
+    return this.props.todos.filter(todo => {
+      if (titleToSearch) {
+        return todo.title.toLowerCase().includes(titleToSearch);
+      }
+
+      return todo;
+    }).filter(todo => {
+      if (completeStatus === 'completed') {
+        return todo.completed;
+      }
+
+      if (completeStatus === 'not completed') {
```

Comment:
Можна було записати використовуючи `switch`
```suggestion
      if (completeStatus === 'completed') {
        return todo.completed;
      }

      if (completeStatus === 'not completed') {
```